tree/ntuple/v7/inc/ROOT/RNTupleView.hxx

```diff
@@ -149,17 +149,40 @@ private:
    FieldT fField;
    /// Used as a Read() destination for fields that are not mappable
    RFieldBase::RValue fValue;
+   /// Whether the user supplied an external memory location to read the value into
+   bool fUserProvidedAddress{false};
```

Comment:
I think we should try to make this a template parameter. Otherwise, reading mappable fields has an additional runtime `if`. In `Bind`, `BindRawPtr`, and `EmplaceNew`, we can then throw a static assert if the template parameter is set.